src/controllers/verifyCommand.ts

```diff
@@ -3,9 +3,11 @@ import { env } from "../typeDefinitions/default.types";
 import { discordTextResponse } from "../utils/discordResponse";
 import { generateUniqueToken } from "../utils/generateUniqueToken";
 import { sendDiscordDm } from "../utils/sendDiscordDm";
+import { storeUniqueToken } from "../utils/storeUniqueToken";
```

Comment:
change the name of function as we are saving more than just a unique token

Reply:
`storeData` works? Can you please suggest a name?

Reply:
I would prefer `sendUserDiscordData`

Reply:
`sendUserDiscordData` seems perfect